components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -88,6 +89,26 @@ export default class AdminSidebar extends React.Component {
         }
 
         this.updateTitle();
+
+        if (!Utils.isMobile()) {
+            $('.admin-sidebar .nav-pills__container').perfectScrollbar({
+                suppressScrollX: true,
```

Comment:
This change does not seem related to this PR, is this needed?

Reply:
Not sure how these came in..Thanks for spotting. Removed 👍